lib/PlacesFeed.jsm

```diff
@@ -90,21 +108,26 @@ class BookmarksObserver extends Observer {
    * @param  {str} uri
    * @param  {str} guid      The unique id of the bookmark
    */
-  onItemRemoved(id, folderId, index, type, uri, guid, parentGuid, source) { // eslint-disable-line max-params
-    if (type === PlacesUtils.bookmarks.TYPE_BOOKMARK &&
-        source !== PlacesUtils.bookmarks.SOURCES.IMPORT &&
-        source !== PlacesUtils.bookmarks.SOURCES.RESTORE &&
-        source !== PlacesUtils.bookmarks.SOURCES.RESTORE_ON_STARTUP &&
-        source !== PlacesUtils.bookmarks.SOURCES.SYNC) {
-      this.dispatch({type: at.PLACES_LINKS_CHANGED});
+
+  onItemRemoved(id, folderId, index, type, uri, guid, parentGuid, source) {
+    // eslint-disable-line max-params
```

Comment:
This also happened for bugzy. Seems like we need to switch to `eslint-disable-next-line` and move the comment.. Although prettier seems to want to insert blank spaces around comments, so moving to the "previous" line might not work either

Reply:
Yeah, I've locally fixed a number of the remaining issues locally by switching to `eslint-disable-next-line`, but there are some conflicts with the `lines-around-comment` stuff yet to be debugged.